src/questions/expression_control.yaml

```diff
@@ -91,4 +91,576 @@
+  answerKeys:
+    - A
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: To terminate the loop
+      key: A
+    - content: To make a jump for next iteration
+      key: B
+    - content: To stop a function's execution
+      key: C
+    - content: It is used to handle errors
+      key: D
+
+- uuid: 23997bf8-e4f1-4ef4-9fa6-c4b8faec5bfe
+  type: SingleChoice
+  content: What is the use of the continue keyword?
+  hint: noHint
+  explanation: The keyword "continue" is mostly used with conditional statements in order to skip the current iteration of a loop.
+  answerKeys:
+    - D
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: It is used to handle errors
+      key: A
+    - content: Used for making a contract object
+      key: B
+    - content: Used for making external calls
+      key: C
+    - content: Used to skip iteration
+      key: D
+
+- uuid: 4b718e63-fb3a-4eff-ae4a-bdfbcc2750b2
+  type: SingleChoice
+  content: Where can we use a return keyword?
+  hint: noHint
+  explanation: The "return" keyword can be used only inside a function or within the body of a function.
+  answerKeys:
+    - C
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: It can be used only inside contructor function
+      key: A
+    - content: It can be defined in for loops without a function
+      key: B
+    - content: Only inside a function
+      key: C
+    - content: None of these
+      key: D
+
+- uuid: 8b75dff0-084f-4b5c-8fe9-1751d4ad35b2
+  type: SingleChoice
+  content: What will happen if the compiler encounters an `break` keyword inside a nested for loop?
+  hint: noHint
+  explanation: When the compiler encounters the `break` keyword, it will stop the current iteration or terminate the loop altogether.
+  answerKeys:
+    - B
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: All the for loops are terminated
+      key: A
+    - content: The for loop in which the break keyword is encountered will be terminated
+      key: B
+    - content: No change will occur
+      key: C
+    - content: None of the above
+      key: D
+
+- uuid: b842c736-1a61-4743-9519-ffd6df8ad24b
+  type: SingleChoice
+  content: In Solidity, is it possible to return a value from a function without using the return keyword?
+  hint: noHint
+  explanation: We can return a value from a function by assigning the return variable to the value we want to return.
+  answerKeys:
+    - A
+  subTopics:
+    - "loops-keywords"
+  difficultyLevel: Low
+  choices:
+    - content: "Yes"
+      key: A
+    - content: "No"
+      key: B
+
+- uuid: 3fa16d55-f4cd-4ab5-bfa6-279353985b77
+  type: SingleChoice
+  content: |
+    Tell whether the following statement is true or false.
+    the data types of returning values should be specified in the function using the keyword `returns`.
```

Comment:
The data types of returning values should be specified in the function using the keyword `returns`.